packages/generator-adp/src/app/questions/configuration.ts

```diff
@@ -0,0 +1,335 @@
+import type { YUIQuestion, GuiOptions, PromptSeverityMessage } from '@sap-ux/inquirer-common';
+import type { ListQuestionOptions } from 'inquirer';
+
+import { validateEmptyString } from '@sap-ux/project-input-validator';
+import { t } from '../../utils/i18n';
+import { AbapProvider, Application, ApplicationManager, EndpointsManager, getEndpointNames } from '@sap-ux/adp-tooling';
+import { getApplicationChoices } from './helper/choices';
+import { showApplicationQuestion, showCredentialQuestion } from './helper/conditions';
+
+/**
+ * Enumeration of prompt names used in the configuration.
+ */
+export enum configPromptNames {
```

Comment:
It might be cleaner to export the enum and interfaces in this file to a separate file(s) constants.ts / types.ts